tests/integration/_experimental/dataset/test_evaluation.py

```diff
@@ -70,7 +70,7 @@ def get_df_inf(n: int = 20) -> pd.DataFrame:
     return pd.DataFrame(records)
 
 
-def get_df_metrics(n: int = 20) -> pd.DataFrame:
+def get_df_mtr(n: int = 20) -> pd.DataFrame:
```

Comment:
Nit: I find this shorthand obscured, any benefit to shorten from `metrics` to `mtr`?

Reply:
this is to distinguish with `df_metrics` from `fetch_evaluation_results`

```sh
datapoints = dp
inferences = inf
metrics = mtr
```